Automattic/wp-calypso
# Stats: force new user to purchase page (still behind feature flag)

Related to #86130 

## Proposed Changes

* Adding a check for the blog creation date to enable new purchase flow for Jetpack sites

## Testing Instructions



* create a JN blog and connect Jetpack
* open the live branch and navigate to the Stats page for the new blog
* the Stats page should load
* apply `stats/checkout-flows-v2` feature flag and verify that you are redirected to a purchase page

## Pre-merge Checklist
